### PR TITLE
DOC:Add example in docstring of load_images_from_dir

### LINKED ISSUE #369: io: Add list example for load_images_from_dir
Each time MaskedMultiSubjectData is called it requires the images of interest to be loaded and masked, otherwise a "StopIteration:" error appears, even if the images have been previously loaded. A helpful optimization would be to allow images to be loaded (load_images) and masked once initially and then to be able to call MaskedMultiSubjectData multiple times thereafter without having to reload/re-mask the images for each call of MaskedMultiSubjectData. A possibility is to update the load_images function so that the object created can be operated on more than once.

### PR DESCRIPTION
**Pull request by sameshl**
closes #369

### CODE REVIEW

**Comment by buildbot-princeton** [comment]
Can one of the admins verify this patch?

**Comment by buildbot-princeton** [comment]
Can one of the admins verify this patch?

**Comment by mihaic** [comment]
Jenkins, OK to test.

File: brainiak/io.py [hunk 1]
```diff
@@ -57,6 +57,10 @@ def load_images_from_dir(in_dir: Union[str, Path], suffix: str = "nii.gz",
     ------
     SpatialImage
         Image.
+
+    Examples
+    --------
+    images = list(io.load_images_from_dir("face_scene", "bet.nii.gz"))
```
Comment (mihaic) [inline brainiak/io.py]: Let's use the leading `>>>` from the doctest format.

**Comment by mihaic** [comment]
Thank you for your contribution, @sameshl.

**Comment by sameshl** [comment]
Glad to help! @mihaic

**Pull request merged**